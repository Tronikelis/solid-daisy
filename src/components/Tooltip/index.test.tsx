import { render } from "@solidjs/testing-library";
import { describe, expect, it } from "vitest";

import { Tooltip } from ".";

describe("<Tooltip />", () => {
    it("renders correctly", () => {
        const { container } = render(() => <Tooltip label="foo">bar</Tooltip>);
        expect(container).toMatchSnapshot();
    });
});
