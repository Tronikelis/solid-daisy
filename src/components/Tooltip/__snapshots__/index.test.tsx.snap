// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Tooltip /> > renders correctly 1`] = `
<div>
  <div
    class="tooltip"
    data-tip="foo"
  >
    bar
  </div>
</div>
`;
